tests/test_initialization.py

```diff
@@ -1,51 +1,32 @@
 import pytest
 
-# pylint: disable=wrong-import-order
 import up42
-from up42.asset import Asset
-from up42.catalog import Catalog
-from up42.job import Job
-from up42.jobcollection import JobCollection
-from up42.jobtask import JobTask
-from up42.order import Order
-from up42.project import Project
-from up42.storage import Storage
-from up42.tasking import Tasking
-from up42.workflow import Workflow
+from up42 import asset, catalog, job, jobcollection, main, order, project, storage, tasking, workflow
 
-from .fixtures.fixtures_globals import (
-    API_HOST,
-    ASSET_ID,
-    JOB_ID,
-    JOBTASK_ID,
-    ORDER_ID,
-    PROJECT_APIKEY,
-    PROJECT_ID,
-    WORKFLOW_ID,
-)
+from .fixtures import fixtures_globals as constants
 
 
 def test_initialize_object_without_auth_raises():
-    up42.main._auth = None
+    main._auth = None  # pylint: disable=protected-access
 
     with pytest.raises(RuntimeError):
-        up42.initialize_project()
+        up42.initialization.initialize_project()
```

Comment:
These changes are not needed

Reply:
And we actually need to check that the signature used by users works

Reply:
And the same below

Reply:
fair point.
If I remove `.initialization`, mypy complains with ` error: Module has no attribute "initialize_project" `

Any suggestions?

Reply:
Proposed below